SU2_CFD/src/SU2_CFD.cpp

```diff
@@ -52,6 +52,17 @@ int main(int argc, char *argv[]) {
   unsigned short nZone;
   char config_file_name[MAX_STRING_SIZE];
   bool fsi, turbo, zone_specific;
+  bool dry_run;
+  std::string filename = "default";
+  
+  /*--- Command line parsing ---*/
+  
+  CLI::App app{"SU2 v6.2.0 \"Falcon\", The Open-Source CFD Code"};
```

Comment:
I added a commandline parser (https://github.com/CLIUtils/CLI11). For the time being the only options are `-d` for the dry run mode and `-h` to show a help message. Let me know what you think!